packages/jsonxt/jsonxt.1.0.0/opam

```diff
@@ -0,0 +1,57 @@
+x-commit-hash: "1436c172d4ecb3a8f60bcc127ff79a6d66a62585"
+opam-version: "2.0"
+synopsis: "Jsonxt - JSON parsers for files, strings and more"
+description: """\
+Jsonxt provides a number of JSON parsers and writers for
+RFC 8259 compliant JSON as well as non-standard extensions
+introduced by Yojson.  Features include
+
+* RFC 8259 compliant when in strict and basic mode
+* Performance focused especially for files and strings
+* Support for standard and extended JSON tree types:
+  * Strict follows a strict interpretation of RFC 8259 with all
+    numbers represented as floats.
+  * Basic extends the strict type to include convenience types while maintaining
+    RFC compliance.  This is compatible with yojson's Basic type
+  * Extended adds additional non-standard types including tuples and variants
+    and is not RFC compliant. This is compatible with yojson's Safe type
+* A number of different parsers including
+  * A standard JSON tree parser for various sources including string, file and channel
+  * A Stream parser that returns a stream of raw JSON tokens.
+  * A monad based parser compatible with async
+* Writers including
+  * File and string writers
+  * A monad based writer that is compatible with async
+  * A stream writer that converts a stream of JSON tokens
+* Support for streaming JSON via Stream.t
+* Standard interfaces including Yojson compatibility"""
+maintainer: "Steve Bleazard <stevebleazard@googlemail.com>"
+authors: "Steve Bleazard <stevebleazard@googlemail.com>"
+license: "MIT"
+homepage: "https://github.com/stevebleazard/ocaml-jsonxt"
+doc: "https://stevebleazard.github.io/ocaml-jsonxt/"
+bug-reports: "https://github.com/stevebleazard/ocaml-jsonxt/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.0" & build}
+  "alcotest" {with-test}
+  "cmdliner" {with-test}
+  "core_kernel" {with-test}
+  "fmt" {with-test}
+  "ppx_sexp_conv" {with-test}
+]
+build: [
+  ["dune" "subst"] {pinned & dev}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```
What's the reason for putting the two?

Reply:
Fixed this in the next release, I am still getting up to speed on the opam/dune interaction as this is my first opam+dune release